src/shape/arc.js

```diff
@@ -35,8 +39,12 @@ extend(ChartInternal.prototype, {
 		$$.radius = $$.radiusExpanded * 0.95;
 		$$.innerRadiusRatio = w ? ($$.radius - w) / $$.radius : 0.6;
 
+		const innerRadius = config.pie_padding ?
```

Comment:
When  `padding` and `innerRadius` options are set, it's ignored the `innerRadius` because of it's prioritizing padding value checking.
```js
pie: {
	padding: 5,
	innerRadius: 20
}
```

For `padding` it's necessary have innerRadius value, so swapping the condition to check innerRadius first, will solve the issue.

I'll do the changes for that.